app/services/tenor.js

```diff
@@ -153,6 +156,13 @@ export default class TenorService extends Service {
                 if (!options.ignoreErrors && !this.error) {
                     this.error = 'Uh-oh! Trouble reaching the Tenor API, please check your connection';
                 }
+
+                if (this.error && this.error.startsWith('API key not valid')) {
+                    // Added an html error field, so that we don't pass raw API errors from tenor to triple-braces in the frontend
+                    this.htmlError = `This version of the Tenor API is no longer supported.<br />
```

Comment:
`htmlError` needs to be marked as a tracked property with `@tracked htmlError = null;` otherwise templates won't react to it's value changing.